.travis.yml

```diff
@@ -46,6 +46,10 @@ jobs:
     - name: Test Suite viewContent&metadata&extensions
       script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
 
+    - stage: Trigger DW
```

Comment:
why digital workspace? this is the content-app project

Reply:
I think it makes sense to trigger a build of DW when its dependency (ACA) changes.
Or how is it possible to check that with any change in ACA, you haven't broken anything in DW?

Reply:
what @popovicsandras is saying is correct

Reply:
do we trigger a build of all the apps when something changes in ADF ?

Reply:
Once we go monorepo we won't have this problem anymore :)

Reply:
No, but ADF is versioned. We trigger builds in ADF every week, when a new beta is coming out.

Reply:
yes we trigger it

Reply:
https://github.com/Alfresco/alfresco-ng2-components/blob/development/.travis.yml#L119

Reply:
Hmm, I was not even aware of that trigger besides the weekly package update. 🤔

Reply:
aca is not on that list https://github.com/Alfresco/alfresco-ng2-components/blob/development/scripts/travis/update/trigger-travis-children.sh#L3 :)

Reply:
it make sense to add it

Reply:
I will do it

Reply:
done